docs/installation.md

```diff
@@ -0,0 +1,89 @@
+## Installation Guidelines :
+
+Before you start following the guidelines, make sure to go through the [prerequisites guide](./prerequisites.md) to install the required tools and packages on your machine.
```

Comment:
### Please remove this line.
Because we haven't seen this file yet. When this file is selected by us in the future, its issue will be created.

Reply:
**Line no. 3 have to be removed. prerequisites guide haven't picked us up yet.**